### ISSUE TITLE
Using multiple tabs

### ISSUE THREAD

**Issue by MaraSedlins** [datacarpentry/spreadsheets-socialsci#79]
Reading over the lesson, I found conflicting a advice about whether it's advisable to use multiple tabs in Excel. In "Formatting data tables in Spreadsheets"/"Keeping track of your analyses", it's suggested to keep notes about your experiment in a new tab to keep the information together with your data. However, when you export an Excel spreadsheet to CSV for preservation/interoperability as recommended in the next section, each tab/sheet will be saved as a separate file. Also, the section about Metadata specifies that metadata should not be contained in the data file itself (experiment notes could be considered a form of descriptive metadata).

I like the suggestion to keep track of experiment notes along with your data, but there could be some clarification about how to manage this information once the dataset/analyses are finalized (e.g. save your notes as a separate CSV or paste them into a .txt file; preserve the link with the dataset by using a strategic file name and/or including information about the dataset at the beginning of the file).

**Comment by chris-prener** [comment]
Thanks for the feedback @MaraSedlins! @VickySteeves and @brownsarahm - since you both have experience with this lesson - do you have thoughts on this issue

**Comment by brownsarahm** [comment]
I agree consistency is better.  However, if you're entering your data using a spreadsheet at all, using excel sheets to separate data from metadata (notes) is a better practice than for example mixingn data and notes in a single sheet.  

My inclination is that close-to-good even if not ideal practices are valuable advice.  Maybe we can make it more clear that we recommend spreadsheet software as an entry tool, but not long term storage.  Using multiple sheets to group things together while collecting is good, then transform to a better system later?

**Comment by MaraSedlins** [comment]
@brownsarahm that makes sense to me!

**Comment by chris-prener** [comment]
I like your proposal @brownsarahm. I think it would be good to give some examples of "so you've got your metadata in the second sheet... now what!?"

**Comment by josenino95** [comment]
I agree with @MaraSedlins that the current state of the lesson contradicts itself about the use of tabs. And @brownsarahm gets to the point in my opinion, about using spreadsheets (and different tabs) for data entry and saving the data to a more universal format that allows long term storage. This contradiction may also arise when we recommend to don't work over the raw data and make a copy of it, possibly in a new tab.

Curiously, I see that the two other spreadsheets lessons (for [Ecologists ](https://datacarpentry.org/spreadsheet-ecology-lesson/01-format-data.html#keeping-track-of-your-analyses) and [Librarians](https://librarycarpentry.org/lc-spreadsheets/01-format-data.html#keeping-track-of-your-analyses)) have taken different approaches to this. Both add screenshots to their lessons, but one embraces the inclusion of notes in a new tab, and the other in a separate file.

In my opinion, the lesson should be more clear on when it is suggested to use tabs and when it is not. I'll try to work on this and talk about it with other maintainers.
